logging/logging/__init__.py

```diff
@@ -191,8 +188,9 @@ def format(self, record):
                 "Style {0} is not supported by logging.".format(self.style)
             )
 
-    def formatTime(self, record, datefmt=None):
-        raise NotImplementedError()
+    def formatTime(self, record):
+        ct = utime.localtime(record.created)
```

Comment:
Keep the datefmt param, just add `assert datefmt is None`. (Or raise NotImplementedError if you want to make it more bytes ;-) ).